digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/pointasset/OracleTrafficSignDao.scala

```diff
@@ -27,7 +27,8 @@ case class PersistedTrafficSign(id: Long, linkId: Long,
                                 validityDirection: Int,
                                 bearing: Option[Int],
                                 linkSource: LinkGeomSource,
-                                expired: Boolean = false) extends PersistedPoint
+                                expired: Boolean = false,
+                                roadSideInfo: Boolean = false) extends PersistedPoint
```

Comment:
this name can be modified too
`invertedDirection: Boolean = false) extends PersistedPoint`